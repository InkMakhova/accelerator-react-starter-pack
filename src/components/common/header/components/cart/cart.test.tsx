import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter as Router, Route } from 'react-router-dom';
import Cart from './cart';

it('should render Cart correctly', () => {
  const {container} = render(
    <Router>
      <Route>
        <Cart />
      </Route>
    </Router>);
  expect(container).toMatchSnapshot();
});
