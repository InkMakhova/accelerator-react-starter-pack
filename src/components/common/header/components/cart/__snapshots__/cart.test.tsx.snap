// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Cart correctly 1`] = `
<div>
  <a
    aria-label="Корзина"
    class="header__cart-link"
    href="/cart"
  >
    <svg
      aria-hidden="true"
      class="header__cart-icon"
      height="14"
      width="14"
    >
      <use
        xlink:href="#icon-basket"
      />
    </svg>
    <span
      class="visually-hidden"
    >
      Перейти в корзину
    </span>
    <span
      class="header__cart-count"
    >
      2
    </span>
  </a>
</div>
`;
